packaging/linux/deb/package_binaries.sh

```diff
@@ -0,0 +1,73 @@
+#! /bin/bash
+
+# Builds the keybase binary and packages it into two ".deb" files, one for i386
+# and one for amd64. The argument to this script is the output directory of a
+# build_binaries.sh build. The package files are created there, in their
+# respective architecture folders.
+#
+# Usage:
+#   ./package_binaries.sh <build_root>
+
+set -e -u -o pipefail
+
+here="$(dirname "$BASH_SOURCE")"
+
+build_root="${1:-}"
+if [ -z "$build_root" ] ; then
+  echo 'Usage:  ./package_binaries.sh <build_root>'
+  exit 1
+fi
+
+version="$(cat "$build_root/VERSION")"
+mode="$(cat "$build_root/MODE")"
+
+name="$("$here/../../binary_name.sh" "$mode")"
+
+if [ "$mode" = "production" ] ; then
+  repo_url="http://dist.keybase.io/linux/deb/repo"
+elif [ "$mode" = "prerelease" ] ; then
+  repo_url="http://s3.amazonaws.com/prerelease.keybase.io/deb"
+elif [ "$mode" = "staging" ] ; then
+  # Note: This doesn't exist yet. But we need to be distinct from the
+  # production URL, because we're moving to a model where we build a clean repo
+  # every time, rather than adding to an existing one. (For S3 compatibility.)
+  repo_url="http://dist.keybase.io/linux/deb_staging/repo"
```

Comment:
fine with me to make this change.  we never made any promises not to change staging releases...